### PR TITLE
enh(UI): Replace loader component

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This replaces the loader component provided by `loaders.css` in favor of Matrial UI skeleton layout.
We display a skeleton layout when overall page (page + header + navigation menu) is loading and when a page content is loading too.

As well, we decreased bundle size by ~45KB by removing `loaders.css` library

https://www.loom.com/share/957e928eec9948889f535f295c450791

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Load the some pages
 -> A skeleton layout ought to appear

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by bdauria** [comment]
I  think the action skeletons would benefit from a reduction in height as the difference with the actual action height is significant.

File: www/front_src/src/components/Hook/index.tsx [hunk 1]
```diff
@@ -44,11 +44,7 @@ const LoadableHooks = ({
 
 const Hook = React.memo(
   (props: Props) => {
-    return (
-      <Suspense fallback={null}>
-        <LoadableHooks {...props} />
-      </Suspense>
-    );
+    return <LoadableHooks {...props} />;
```
Comment (bdauria) [inline www/front_src/src/components/Hook/index.tsx:47]: What happens here, when a module such as BAM is installed, the page is refreshed and the top counter hook is not yet loaded?

**Review by bdauria — DISMISSED** [review]

**Comment by kduret** [comment]
I'm not sure this is the best way to add a full skeleton for header bar.
Because it adds a kind of clipping when skeleton disappears (content skeleton jumps top before header bar appears)

https://www.loom.com/share/638fedf644c64f0681393bd80a67ac2a

**Review by kduret — COMMENTED** [review]
I'm not sure this is the best way to add a full skeleton for header bar.
Because it adds a kind of clipping when skeleton disappears (content skeleton jumps top before header bar appears)

https://www.loom.com/share/638fedf644c64f0681393bd80a67ac2a

**Comment by Thebarda** [comment]
> I'm not sure this is the best way to add a full skeleton for header bar.
> Because it adds a kind of clipping when skeleton disappears (content skeleton jumps top before header bar appears)
> 
> 
> https://www.loom.com/share/638fedf644c64f0681393bd80a67ac2a

@kduret Fixed ! It's much more satisfying https://www.loom.com/share/52823c2f6d094d8d957f1f8fef3c1113 👀

**Review by bdauria — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]
LGTM

IMHO we'll may need to remove the first grey skeleton of header : 
step 1 : 
image
step 2 : 
image
step 3 : 
image

I think step 1 for header bar is not necessary

**Comment by bdauria** [comment]
> LGTM
> 
> IMHO we'll may need to remove the first grey skeleton of header :
> step 1 :
> image
> step 2 :
> image
> step 3 :
> image
> 
> I think step 1 for header bar is not necessary

Actually it’s necessary so that the layout remains "stable" (there is no vertical shift when the header appears)

**Review by kduret — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**